dart/dynamics/SimpleFrame.cpp

```diff
@@ -120,6 +132,13 @@ SimpleFrame& SimpleFrame::operator=(const SimpleFrame& _otherFrame)
 }
 
 //==============================================================================
+std::shared_ptr<SimpleFrame> SimpleFrame::createChildSimpleFrame(
```

Comment:
I am strongly against (2). If you want to create a kinematic tree, then you should create a `Skeleton`. I see no point in duplicating that functionality here.